MiniScopeControlDlg.cpp

```diff
@@ -619,27 +619,27 @@ void CMiniScopeControlDlg::OnBnClickedRecord()
 	CTime time = CTime::GetCurrentTime();
 	std::string tempString;
 
-	UpdateData(TRUE);
```

Comment:
Hey Phil, Are you sure you meant to remove the call to 'UpdateData(TRUE)'? I think this call is needed to updated the mMouseName member variable with any changes in the Mouse Name text box.

Reply:
I couldn't recall why I did this.... Somehow the name still updates and works so maybe MFC textbox is self-updating. But you are right this is not necessary